daml-lf/governance.rst

```diff
@@ -55,11 +55,10 @@ Overview
   transactions.  Moreover, a one-line summary of each version must be
   present in the ``daml-lf-X.proto`` and ``transaction.proto`` files.
 
-* Every change to Daml-LF values, nodes, transactions or languages
-  *must* be checked for governance rules conformance and be approved
-  by at least two persons among Rémy Haemmerle, Martin Huschenbett
-  and Sophia Mota. Proposed changes can be filed as tickets under
-  labels "component/daml-lf" and "discussion".
+* Every change to Daml-LF values, nodes, transactions or languages *must* be
+  checked for governance rules conformance and be approved by Rémy Haemmerle
+  and Sofia Faro. Proposed changes can be filed as tickets under labels
```

Comment:
```suggestion
  checked for governance rules conformance and be approved at least two persons among by Rémy Haemmerle (@remyhaemmerle-da), Sofia Faro (@sofiafaro-da) and Moritz Kiefer (@cocreature). Proposed changes can be filed as tickets under labels
```

Not at all saying that @remyhaemmerle-da and @sofiafaro-da aren’t sufficient and if they are available they should be involved, but this reduces the bus factor a bit and avoids deadlocks when someone is on vacation or unavailable for other reasons.

Reply:
I agree with @cocreature

Reply:
Agreed :)